src/tableutils.js

```diff
@@ -569,7 +569,9 @@ function createEmptyRows( writer, table, insertAt, rows, tableCellToInsert, attr
 // @param {module:engine/model/position~Position} insertPosition
 function createCells( cells, writer, insertPosition, attributes = {} ) {
 	for ( let i = 0; i < cells; i++ ) {
-		writer.insertElement( 'tableCell', attributes, insertPosition );
+		const tableCell = writer.createElement( 'tableCell', attributes );
+		writer.insert( tableCell, insertPosition );
```

Comment:
You can insert `tableCell` to the `document` at the end (after inserting `paragraph` to the `tableCell`). This will create one less document operation.